.gitignore

```diff
@@ -15,5 +15,5 @@
 
 /node_modules
 /yarn-error.log
-
+.idea
```

Comment:
More for my education: What are these `.idea` files?

Reply:
IDE files specific to IDEA (RubyMine) (.idea is a directory)